packages/shared/src/components/sidebar/SidebarBottomSection.tsx

```diff
@@ -29,6 +44,9 @@ export function SidebarBottomSectionSection({
       icon: () => <ListIcon Icon={() => <TerminalIcon />} />,
       title: 'Changelog',
       path: `${process.env.NEXT_PUBLIC_WEBAPP_URL}sources/daily_updates`,
+      badge: {
+        active: isChangelogModalActive,
+      },
```

Comment:
We actually support `alert` already on the item.

Reply:
As far as I can see it alert is used on the icon in this case. (also positioned in the icon container) So I think new prop is required for this, I will however copy the signature by using element type instead of object, I'll like it more in this case and because render props is obviously used in the rest of the component.

Reply:
after some though using existing `rightIcon` prop